hirosystems/stacks-blockchain-api/templates/api-writer/statefulset.yaml

```diff
@@ -288,8 +290,6 @@ spec:
           args:
           - "./lib/index.js"
           - "from-parquet-events"
-          - {{ printf "--workers=%d" (int .Values.apiWriter.config.replayEvents.workers) | quote }}
```

Comment:
Looking back at my working configuration, there definitely is a `--workers` argument. Otherwise, I believe its default to `1` which will not have much improvement

Reply:
@csgui Any guidance here is greatly appreciated! 🙏

Reply:
Hey guys, sorry for the late reply. That has skipped me.

@deantchi is right. The default value for `workers` is `1`.

And also @CharlieC3 is right. There is an issue with the implementation. It's not taking into consideration the name of the `workers` argument. It can be anything at the moment. 

This [line](https://github.com/hirosystems/stacks-blockchain-api/blob/develop/src/event-replay/parquet-based/helpers.ts#L74) is fetching the `workers` argument from the command line.

Nevertheless, this problem does not hinder an event-replay run. The `workers` argument (or any other name at the moment) is need for the number of parallel processes that are spawned during the event-replay. If not present will default to `1`.

I've opened an [issue](https://github.com/hirosystems/stacks-blockchain-api/issues/1773) to improve the arguments handling.

Reply:
Thanks @csgui!

@deantchi I've added back the parquet workers arg.